docs/specification-ligo.md

```diff
@@ -873,3 +873,25 @@ Motivation:
 
 The deployment of contract with metadata has an extra step: a dedicated contract for carrying metadata has to be originated first.
 Then the baseDAO contract should include the reference to a metadata key in the contract in order to be compliant with TZIP-16.
+
+## Example contracts
+
+### RegistryDAO
+
+The storage, including configuration for the RegistryDAO
+[specification](https://github.com/tqtezos/baseDAO/blob/master/docs/registry.md)
+is included in `ligo/src/registryDAO.mligo`. You can convert this into a
+Michelson expression during the BaseDAO origination using the `ligo
+compile-storage` command as follows.
+
+
+```
+ligo compile-storage ligo/src/registryDAO.mligo dummy 'default_registry_DAO_full_storage(("tz1QozfhaUW4wLnohDo6yiBUmh7cPCSXE9Af" : address), ("tz1QozfhaUW4wLnohDo6yiBUmh7cPCSXE9Af" : address), 0n, 0n, 0n, 0n, 0n)'
+```
+
+The `default_registry_DAO_full_storage` is a LIGO function defined in
+`ligo/src/RegistryDAO.mligo`, which returns a LIGO expression for storage,
+which is converted into Michelson using the `compile-storage` command. The
+`dummy` entrypoint is just included to make this command work. The arguments to
+this function are the admin address, the token address and the configuration
+parameters described in the Registry spec, which are 'a', 'b', 's_max', 'c' and 'd'.
```

Comment:
I think this should be in `ligo/README.md` instead of `specification-ligo.md`.

Reply:
Ah, yes. Btw there is already some mention of storage generation in that file. Do you think if we should amend it, point to the RegistryDAO section maybe?

https://github.com/tqtezos/baseDAO/blob/master/ligo/README.md#generating-storage

Reply:
Yeah, I'm good with that. Maybe we could update it to something like this or similar:

```
Currently we don't provide any convenient way to generate `BaseDAO` storage, but you can use [RegistryDAO Storage generation](link-to-section) as example.
```

Reply:
Done.